.gitignore

```diff
@@ -11,4 +11,5 @@ __pycache__/
 /profiles/migrations/
 /node_modules
 package-lock.json
-package.json
\ No newline at end of file
+package.json
```

Comment:
Why?

Reply:
To be honest, I don’t know why this line is hanging, I just wanted to add /FrontEnd/node_modules/ ..

Reply:
this file has already removed from gitignore in develop branch and I merged this changes, so now it is tracked